src/host/host.cpp

```diff
@@ -97,7 +97,7 @@ int main(int argc, char *argv[]) {
         std::shared_ptr<ThreadFactory> threadFactory = 
             std::shared_ptr<ThreadFactory>(new ThreadFactory());
         std::shared_ptr<ThreadManager> threadManager = 
-            ThreadManager::newSimpleThreadManager(8);
+            ThreadManager::newSimpleThreadManager(64);
```

Comment:
Set this thread number and `NumTCS` to 8 or 16. Correct the math in the comment in `ortoa.conf` and then I'm good to merge this